python/templates/Collection.h.jinja2

```diff
@@ -203,8 +203,8 @@ private:
   bool m_isValid{false};
   mutable bool m_isPrepared{false};
   bool m_isSubsetColl{false};
-  uint32_t m_collectionID{0};
-  mutable std::unique_ptr<std::mutex> m_storageMtx{nullptr};
+  uint32_t m_collectionID{static_cast<uint32_t>(podio::ObjectID::untracked)};
+  mutable std::unique_ptr<std::mutex> m_storageMtx{std::make_unique<std::mutex>()};
```

Comment:
I think we had this construct at some point and moved to the initialization in the member initializer list due to some nvcc issues in k4clue. https://github.com/AIDASoft/podio/pull/333 

Could you verify that nvcc can now handle this?

Reply:
CUDA stuff was removed from k4Clue in https://github.com/key4hep/k4Clue/commit/13709a97ee286d9420d2789db05cf4016e1ae698 so it doesn't use it anymore.